static/js/components/subject_page/block.tsx

```diff
@@ -200,11 +200,7 @@ function renderTiles(
     const place = tile.placeDcidOverride
```

Comment:
To confirm, does line tile in nodejs support child-type?  I know web-component does (https://github.com/datacommonsorg/website/blob/9aad7497955e20b180c1c661b2ef57986a096d0e/packages/web-components/examples/all-charts.html#L328-L333)

Reply:
it does not .. do we want to enable it?

I chose not to enable because:
1. line tile doesn't set max number of places so the number of lines can get crazy
2. the legend can't be hovered so if there are lots of lines, don't have a way to tell which line is which

Reply:
Got it.  I think this is fine for chart generation.

I think we should do (1), given web-components support setting childPlaceType?

You mean (2) doesn't work only for the SVGs right, but the tile supports it:
<img width="1325" alt="Screenshot 2023-09-27 at 4 27 06 PM" src="https://github.com/datacommonsorg/website/assets/4375037/abf9b7f6-4b8b-44ad-ba5f-e8e577c1ead0">

Reply:
yeup, agreed that we should do (1) & yes, (2) only applies to node server svgs